src/routes/router.ts

```diff
@@ -63,17 +63,19 @@ RootRouter.use("/microscope/dlq", MicroscopeDlqRouter);
 
 // Maintenance mode - needs to be before all other routes
 RootRouter.use(MaintenanceRouter);
+//
+// RootRouter.get("/create-branch-options", cookieSessionMiddleware, maybeJiraSymmetricJwtMiddleware, GithubCreateBranchOptionsGet);
 
 // Session redirect
 RootRouter.get(["/session", "/session/*"], SessionGet);
 
 RootRouter.use(cookieSessionMiddleware);
 
+RootRouter.get("/create-branch-options", maybeJiraSymmetricJwtMiddleware, GithubCreateBranchOptionsGet);
```

Comment:
The git different show it moves up to above the "/github-manifest". Just double check there's no real intention for "moving up" right?

Reply:
It was moved from line 65ish were those comments i left behind were